src/builtins/register.rs

```diff
@@ -153,23 +152,27 @@ pub async fn register_application_commands_buttons<U, E>(
                             b.custom_id("register.guild")
                                 .label("Register in guild")
                                 .style(serenity::ButtonStyle::Primary)
+                                .emoji(serenity::ReactionType::Unicode("📋".to_string()))
```

Comment:
`.emoji()` takes `impl Into<ReactionType>`, so any type that implements the `Into<ReactionType>` trait. As you can see [here](https://docs.rs/serenity/latest/serenity/model/channel/enum.ReactionType.html#impl-From%3Cchar%3E-for-ReactionType), this includes `char`. So you can just write `.emoji('📋')`